### PR TITLE
[WebProfilerBundle][logger] added missing deprecation message.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | https://github.com/symfony/symfony/pull/14696#issuecomment-105710882 |
| Tests pass? | yes |
| License | MIT |

before:

selection_012

after:

selection_011

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig [hunk 1]
```diff
@@ -110,6 +110,8 @@
 
 
 {% macro display_message(log_index, log, is_deprecation) %}
+    {{ is_deprecation ? 'DEPRECATED':log.priorityName }} - {{ log.message }}
```
Comment (jakzal) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig]: I might be picky, but can you add spaces around `:` to make it a bit more readable? :)

Comment (aitboudad) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig]: done :)

**Comment by stof** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @aitboudad.

**Pull request merged**